src/date_input.jsx

```diff
@@ -95,24 +95,12 @@ var DateInput = React.createClass({
   },
 
   render () {
-    const {
```

Comment:
@ruianderson Wouldn't removing this hunk cause other warnings for unknown props?